vere/pier.c

```diff
@@ -408,7 +408,13 @@ _pier_work_send(u3_writ* wit_u)
   u3_newt_write(&god_u->inn_u, u3k(wit_u->mat), wit_u);
 }
 
-/* u3_pier_work_save(): tell worker to save checkpoint.
+/*
+  u3_pier_work_save(): tell worker to save checkpoint.
+
+  If there are no unsnapshotted events, then do nothing, otherwise, send
+  `[%save ~]` to the slave.
+
+  XX Should we report success before we update `pir_u->sav_u->ent_d`?
```

Comment:
Upon further consideration, this comment would be clearer if it was something like: "Should we wait for some report of success before updating ..."